### PR TITLE
refactors butchering of humans with chainsaw/meatcleaver

### PR DESCRIPTION
**Pull request by hal9000PR**
## What Does This PR Do
makes butchering humans with a syndie chainsaw/meatcleaver use traits instead of a direct istype check

examining these items will also show what they do.

## Why It's Good For The Game
Traits are good. istype checks are bad, makes it easier to tell what is happening with the code.

also, bonus adminbus

## Images of changes
image

(humans was changed to people)
## Changelog
🆑 
add: admins can now adminbus any item to butcher dead corpses.
/:cl:

### CODE REVIEW

**Comment by hal9000PR** [comment]
I am incapable of writing.

**Review by SabreML — APPROVED** [review]

File: code/game/objects/items.dm [hunk 1]
```diff
@@ -230,6 +230,9 @@ GLOBAL_DATUM_INIT(fire_overlay, /image, image("icon" = 'icons/goonstation/effect
 		msg += "*--------*"
 		. += msg
 
+	if(HAS_TRAIT(src, TRAIT_BUTCHERS_HUMANS))
+		. += "<span class='warning'> Can be used to butcher dead humans into meat while on harm intent. </span>"
```
Comment (SteelSlayer) [inline code/game/objects/items.dm:234]:
```suggestion
		. += "<span class='warning'>Can be used to butcher dead humans into meat while on harm intent.</span>"
```

**Review by SteelSlayer — APPROVED** [review]
LGTM other than the nitpick.

**Comment by Wojz** [comment]
Would be probably better to say "people" instead of "humans" since most of these are only "humans" in code.

**Review by dearmochi — APPROVED** [review]

**Pull request merged**